src/Deserialization/audio.jl

```diff
@@ -2,7 +2,7 @@ function deserialize_audio_summary(summary)
     audio = summary.audio
 
     if audio.content_type == "audio/wav"
-        value = load(Stream(format"WAV", PipeBuffer(audio.encoded_audio_string)))
+        value = load(Stream{format"WAV"}(PipeBuffer(audio.encoded_audio_string)))
```

Comment:
This format requires FileIO >1.6.0 😆

Reply:
My patch to this is

```julia
if isdefined(FileIO, :action)
    # FileIO >= 1.6
    _format_stream(format, io) = FileIO.Stream{format}(io)
else
    _format_stream(format, io) = FileIO.Stream(format, io)
end
```

Reply:
thanks!

Reply:
would it make sense to support only FileIO>=1.6 ? Or that is too recent?

Reply:
Your call, I personally prefer to add a compat patch to avoid unnecessary version resolving locks.